priv/kafka.bnf

```diff
@@ -250,3 +250,9 @@ DescribeGroupsResponse => [GroupDescription]
     MemberMetadata => bytes
     MemberAssignment => bytes
 
+SaslHandshakeRequest => Mechanism
+  Mechanism => string
+
+SaslHandshakeResponse => ErrorCode [EnabledMechanism]
+  ErrorCode => int16
+  EnabledMechanism => string
```

Comment:
It's "enabled mechanisms" - "Array of mechanisms enabled in the server". See SaslHandshake API in https://kafka.apache.org/protocol.html.